openvm/src/lib.rs

```diff
@@ -290,15 +290,6 @@ pub fn compile_openvm(
     guest: &str,
     guest_opts: GuestOptions,
 ) -> Result<OriginalCompiledProgram, Box<dyn std::error::Error>> {
-    // wrap the sdk config (with the standard extensions) in our custom config (with our custom extension)
-    let sdk_vm_config = SdkVmConfig::builder()
-        .system(Default::default())
-        .rv32i(Default::default())
-        .rv32m(Default::default())
-        .io(Default::default())
-        .keccak(Default::default())
```

Comment:
Won't the tests fail if we remove keccak from the default list?

Reply:
Now I'm kinda confused because the tests do pass but the keccak guest doesn't have the openvm.toml file

Reply:
Lower down it uses a default one if the file is not provided. Could we make it a hard error instead?

Reply:
I kept the default because thats also the behaviour of openvm, that way programs that don't use the specialized precompiles (e.g., keccak, curves...) don't need the openvm.toml

Reply:
> Lower down it uses a default one if the file is not provided. Could we make it a hard error instead?

yes but the default one doesn't have Keccak